docs/src/pages/customization/css-in-js/css-in-js.md

```diff
@@ -273,7 +273,7 @@ Use the function signature if you need to have access to the theme. It's provide
   - `options.name` (*String* [optional]): The name of the style sheet. Useful for debugging.
     If the value isn't provided, it will try to fallback to the name of the component.
   - `options.flip` (*Boolean* [optional]): When set to `false`, this sheet will opt-out the `rtl` transformation. When set to `true`, the styles are inversed. When set to `null`, it follows `theme.direction`.
-  - The other keys are forwarded to the options argument of [jss.createStyleSheet([styles], [options])](http://cssinjs.org/js-api/#create-style-sheet).
+  - The other keys are forwarded to the options argument of [jss.createStyleSheet([styles], [options])](https://cssinjs.org/jss-api?v=v10.0.0-alpha.7#create-style-sheet).
```

Comment:
This works as well: https://cssinjs.org/jss-api#create-style-sheet

Reply:
@joshwooding probably best to be version agnostic. Making that change now.